analyzer/src/funTest/assets/projects/synthetic/pipenv/Pipfile

```diff
@@ -14,4 +14,4 @@ MarkupSafe = "==1.0"
 Werkzeug = "==0.15.3"
 
 [requires]
-python_version = "2.7"
+python_version = "2"
```

Comment:
Out of curiosity, I was doing some research whether this field really doesn't support any version ranges. At first I found [this](https://pipenv-fork.readthedocs.io/en/latest/basics.html#general-recommendations-version-control) which states "python_version should be in the format X.Y", so I'm surprised this actually works. Then I found [this](https://github.com/pypa/pipenv/issues/1050) which is really fun to read, and reaffirms my view that the pipenv maintainer is a really odd guy :wink:

Reply:
Yes, I found this discussion as well when looking for a solution, it was interesting. Using only the major is not documented anywhere, but as it's working I think it's ok to use it for the test.

Reply:
> Using only the major is not documented anywhere, but as it's working

Could you maybe add this part to the commit message?

Reply:
Done.